'''
The assert can take a second argument that is passed to the AssertionError
 raised if the assertion fails. 
'''
temp = -10
assert (temp >= 0), 'Mais frio que Zero graus'
